src/senaite/patient/subscribers/configure.zcml

```diff
@@ -22,13 +22,6 @@
       handler=".analysisrequest.on_object_created"
       />
 
-  <!-- Before transition event -->
-  <subscriber
-      for="senaite.patient.content.patient.IPatient
-           Products.DCWorkflow.interfaces.IBeforeTransitionEvent"
-      handler=".patient.on_before_transition"
-      />
```

Comment:
At the time of writing, I thought this subscriber was kind of smart. But I agree to remove it to avoid unneccessary brain twists and that it is probably just confusing instead.

Reply:
Since mrn must be unique regardless of patient status, this subscriber is no longer needed